src/main/resources/templates/clients.html

```diff
@@ -56,7 +56,6 @@ <h5>Project chart, budget and more details will show here</h5>
 </div>
 
 </body>
-</html>
```

Comment:
There was an unnecessary </html> tag in the clients.html file, producing an error when navigating to it. I went ahead and removed it.